src/app/public/testing/infinite-scroll/infinite-scroll-fixture.ts

```diff
@@ -0,0 +1,48 @@
+import {
+  DebugElement
+} from '@angular/core';
+import {ComponentFixture} from '@angular/core/testing';
+import {SkyAppTestUtility} from '@skyux-sdk/testing';
+
+/**
+ * Provides information for and interaction with a SKY UX infinite scroll component.
+ * By using the fixture API, a test insulates itself against updates to the internals
+ * of a component, such as changing its DOM structure.
+ */
+export class SkyInfiniteScrollFixture {
+  private debugElement: DebugElement;
+  private skyBtnSelector = 'div.sky-infinite-scroll > button.sky-btn';
+
+  constructor(
+    private fixture: ComponentFixture<any>,
+    skyTestId: string
+  ) {
+    this.debugElement = SkyAppTestUtility.getDebugElementByTestId(fixture, skyTestId, 'sky-infinite-scroll');
+
+    this.waitForComponentInitialization();
+  }
+
+  public isEnabled(): boolean {
+    return !!this.debugElement.componentInstance.enabled;
+  }
+
+  public isButtonAvailable() {
+    return this.debugElement.nativeElement.querySelector(this.skyBtnSelector) instanceof HTMLButtonElement;
+  }
+
+  public clickLoadMoreButton() {
```

Comment:
Don't forget to add return types to all your methods `public clickLoadMoreButton():void {`.